app/views/burgers/new.html.erb

```diff
@@ -29,5 +29,5 @@
     <%= f.text_area :image_url %>
   </div>
 
-  <%= f.submit "Add Burger" %>
+  <button><%= f.submit "Add Burger" %></button>
```

Comment:
Rather than wrapping this in a `<button>` element, you could utilize the `f.button` Rails form builder.